target/linux/mvebu/base-files/etc/uci-defaults/04_mambafan

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+#
+# Copyright (C) 2017 LEDE-Project.org
+#
+ 
+. /lib/mvebu.sh
+ 
+board=$(mvebu_board_name)
+ 
+case "$board" in
+armada-xp-linksys-mamba)
+
+	cat > /etc/init.d/fan_control << 'FC_EOF'
+#!/bin/sh /etc/rc.common
+
+# mambafan daemon control script (/etc/init.d/fan_control)
+# for start/stop of mamba fan monitor (/usr/sbin/fan_monitor)
+
+START=99
```

Comment:
I think that START value should be below 95 so that your script runs before S95done (that should run last as it runs /etc/rc.local that is claimed to be run after the system init finishes).

Fan control is not dependent on network or anything else that could start slowly, so there is no need to push the start very late. It could well be 45 or 55 or something.

(I know that there are some scripts running later, but that is bad practice and against the documentation concerning /etc/rc.local )